extensions/azurehybridtoolkit/package.nls.json

```diff
@@ -0,0 +1,5 @@
+{
+	"displayName": "Azure SQL Hybrid Cloud Toolkit Notebook Extension",
+	"description": "Opens up Azure SQL Hybrid Cloud Toolkit notebook",
+	"title.openNotebookFolder": "Open Azure SQL Hybrid Cloud Toolkit notebook"
```

Comment:
This is a Jupyter book - not a single Notebook. Make sure the strings are all accurate in that respect